os/mounting-storage.md

```diff
@@ -16,9 +16,13 @@ coreos:
         Type=ext3
 ```
 
-As you can see, it's pretty simple. You specify the attached device and where you want it mounted. Optionally, you can provide a file system type.
+[Cloud-Config][cloud-config] will store `media-ephemeral.mount` unit file into `/etc/systemd/system` directory and systemd will catch it on boot. As you can see, it's pretty simple. You specify the attached device and where you want it mounted. Optionally, you can provide a file system type.
```

Comment:
"Cloud-config then writes a unit file named `media-ephemeral.mount` beneath the `/etc/systemd/system` directory. Mount units describe the source filesystem and target mount point. *Systemd* mounts filesystems defined in such units at boot time.